stream_alert/athena_partition_refresh/main.py

```diff
@@ -482,15 +512,22 @@ def handler(*_):
 
     # Initialize the SQS client and recieve messages
     stream_alert_sqs = StreamAlertSQSClient(config)
-    stream_alert_sqs.get_messages()
+    # Get the first batch of messages from SQS.  If there are no
+    # messages, this will exit early.
+    stream_alert_sqs.get_messages(max_tries=2)
 
     if not stream_alert_sqs.received_messages:
-        LOGGER.info('No messages recieved, exiting')
+        LOGGER.info('No SQS messages recieved, exiting')
         return
 
+    # If the max amount of messages was initially returned,
+    # then get the next batch of messages
+    if len(stream_alert_sqs.received_messages) == 20:
```

Comment:
@jacknagz it's incredibly unclear where this value comes from though - a named constant (or even a property) would help a ton here, and it should be clear that it's a multiple of `max_message * max_tries` Also why do you perform `get_messages` with `max_tries=2` once and then `max_tries=8` next?